lib/tests/change-federation.js

```diff
@@ -223,6 +257,191 @@ const execute = (description, newFederationConfig) => {
+    
+            if(!isLovellActive) {
+                return this.skip();
+            }
+
+            const bridgeState = await getBridgeState(rskTxHelper.getClient());
+    
+            expect(bridgeState.pegoutsWaitingForConfirmations.length).to.be.equal(0, 'No pegout should be waiting for confirmations.');
+            expect(bridgeState.pegoutsWaitingForSignatures.length).to.be.equal(0, 'No pegout should be waiting for signatures.');
+    
+            await assertSvpValuesNotPresentInStorage(rskTxHelper);
+    
+        });
+    
+        it('should create the SVP Fund transaction on the next updateCollections call', async function () {
+    
+            if(!isLovellActive) {
+                return this.skip();
+            }
+
+            // Act
+    
+            const initialBridgeState = await getBridgeState(rskTxHelper.getClient());
+    
+            await rskUtils.waitAndUpdateBridge(rskTxHelper);
+    
+            const finalBridgeState = await getBridgeState(rskTxHelper.getClient());
+    
+            // Assert
+    
+            // The SVP fund transaction should be created and put in waiting for confirmations.
+            expect(finalBridgeState.pegoutsWaitingForConfirmations.length).to.be.equal(1, 'There should be one pegout waiting for confirmations.');
+            expect(finalBridgeState.pegoutsWaitingForSignatures.length).to.be.equal(0, 'No pegout should be waiting for signatures.');
+    
+            const svpFundTxWaitingForConfirmations = finalBridgeState.pegoutsWaitingForConfirmations[0];
+    
+            const pegoutCreationBlockNumber = Number(svpFundTxWaitingForConfirmations.pegoutCreationBlockNumber);
+    
+            const expectedPegoutCreationBlockNumber = commitFederationCreationBlockNumber + 1;
+            expect(pegoutCreationBlockNumber).to.be.equal(expectedPegoutCreationBlockNumber, 'The svp fund tx pegout creation block number should be the block that contains the first updateCollections call right after the commitFederation call.');
+    
+            const rawSvpFundTransaction = svpFundTxWaitingForConfirmations.btcRawTx;
+    
+            const svpFundTransaction = bitcoinJsLib.Transaction.fromHex(rawSvpFundTransaction);
+    
+            expect(svpFundTransaction.outs.length).to.be.equal(3, 'The SVP fund transaction should have 3 outputs.');
+    
+            const proposedFederationAddress = await bridge.methods.getProposedFederationAddress().call();
+    
+            // The output addresses should be in the expected order.
+            const proposedFederationOutput = svpFundTransaction.outs[0];
+            const proposedFederationFlyoverOutput = svpFundTransaction.outs[1];
+            const activeFederationOutput = svpFundTransaction.outs[2];
+    
+            const actualProposedFederationAddress = bitcoinJsLib.address.fromOutputScript(proposedFederationOutput.script, btcTxHelper.btcConfig.network);
```

Comment:
```suggestion
            const proposedFederationAddressFromOutputScript = bitcoinJsLib.address.fromOutputScript(proposedFederationOutput.script, btcTxHelper.btcConfig.network);
```

wdyt? just a suggestion

Reply:
Updated.